jbpm-runtime-manager/src/test/resources/jbpm.audit.jms.properties

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
is this intentionally left empty? if so maybe just add a comment about it so it won't be removed by accident